src/client/features/view/components/tooltips/formatArray.js

```diff
@@ -0,0 +1,24 @@
+  //Push a metadata field to the bottom of the list
+  function pushToBottom(data, field) {
```

Comment:
I have a suspicion that this can be simplified with lodash.

Can you read over the lodash documentation to see if you can find a function that can do something similar?